### DISCUSSION TITLE
⏳ 🔗 Breadcrumbs widget

### DISCUSSION THREAD

**Discussion by ZoltanGalCMDesign** [elementor/elementor#21147] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

At this moment (as I know it) I need to install additional plugin to display breadcrumbs on a website using elementor: https://elementor.com/help/breadcrumbs-widget-pro/

### Describe the solution you'd like

It would be nice to have a breadcrumbs widget in Elementor core without the need of an extra WP plugin. Thanks.

### Describe alternatives you've considered

Used 3rd party plugins

**Comment by danielvigh** [comment]
+1 :)

**Comment by 79ho3ein** [comment]
+1
Also three other suggestions that help with this feature.

1- Possibility of custom name for the first position. **home** > blog > blog post name.

2- Custom icon for divider in breadcrumbs

3- breadcrumbs horizontal Scroll in mobile

**Comment by loucasl** [comment]
Since the breadcrumb feature typically comes with SEO plugins, which in my opinion are mandatory to have on all websites anyway, is there really a need for this to be in Elementor?

**Reply by 79ho3ein** [reply]
I think yes. It would be much better if Elementor had its own widget for breadcrumbs. Because Elementor will definitely offer much better and wider customization. Most importantly, many users use the free version of RankMeth, which does not offer any widgets for breadcrumbs. This is where Elementor users will feel a strong need for this widget.

**Reply by bazeboiee** [reply]
Also not all SEO packages offer breadcrumbs widgets and why would you want to be constrained to one type of SEO plugin just to get breadcrumbs? It's not their job to provide that functionality even if some of them do.

**Comment by bazeboiee** [comment]
I woudl strongly urge the Elementor crew to have this as an option so we don't have to rely on 3rd party plugins to attain this funcitonality. It shouldn't be up to us to find another plugin to add what you would assume is a fairly crucial part of SEO for websites.
